app/features/settings/components/WindowAlwaysOnTopToggle.js

```diff
@@ -0,0 +1,118 @@
+// @flow
+
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import type { Dispatch } from 'redux';
+
+import { HelpText, TogglesContainer } from '../styled';
+import {
+    setWindowAlwaysOnTop
+} from '../actions';
+
+import ToggleWithLabel from './ToggleWithLabel';
+
+type Props = {
+
+    /**
+     * Redux dispatch.
+     */
+    dispatch: Dispatch<*>;
+
+    /**
+     * Start with Window Always on top value in (redux) state.
+     */
+    _windowAlwaysOnTop: boolean;
+};
+
+type State = {
+
+    /**
+     * Start with Window Always on top value in (local) state.
+     */
+    windowAlwaysOnTop: boolean;
+};
+
+/**
+ * Window always open on top placed in Settings Drawer.
+ */
+class WindowAlwaysOnTopToggle extends Component<Props, State> {
+    /**
+     * Initializes a new {@code WindowAlwaysOnTopToggle} instance.
+     *
+     * @inheritdoc
+     */
+    constructor(props) {
+        super(props);
+
+        this.state = {
+            windowAlwaysOnTop: true
```

Comment:
You don't need to store state here, just use the prop value.

Reply:
i will, but i was following style of [settings components](https://github.com/jitsi/jitsi-meet-electron/blob/master/app/features/settings/components/StartMutedToggles.js#L55)

Reply:
@saghul 
I think this was done because of `isDefaultChecked` prop. Don't remember what problem was it causing.

Reply:
Oh, right!